src/foam/nanos/crunch/rules.jrl

```diff
@@ -28,21 +28,25 @@ p({
   },
   "lifecycleState":1
 })
-p({
-  "class":"foam.nanos.ruler.Rule",
-  "id":"68afcf0c-c718-98f8-0841-75e9-crunch0",
-  "name":"Notify User On Top Level Capability Status Update",
-  "priority":100,
-  "ruleGroup":"crunch",
-  "documentation":"send notification to user on every status update for their visible capabilities",
-  "daoKey":"userCapabilityJunctionDAO",
-  "operation":3,
-  "after":true,
-  "enabled":true,
```

Comment:
making this rule enabled = false gives us the versatility of updating the action and then just enabling in the rule. The concern with commenting out the rule is potential journal conflicts in the future.